cardano-api/src/Cardano/Api/Query.hs

```diff
@@ -23,35 +23,40 @@ module Cardano.Api.Query (
     -- * Internal conversion functions
     toConsensusQuery,
     fromConsensusQueryResult,
+
+    -- TODO: Move me
+    LedgerState(..),
+    ProtocolState(..),
```

Comment:
Not sure where to move these. Perhaps under `Cardano.Api.Ledger`? Simply because the underlying types are:

* `Shelley.Spec.Ledger.LedgerState` `NewEpochState`
* `Shelley.Spec.Ledger.API.Protocol` `ChainDepState`

Reply:
I think we should leave them here for now under the `Wrapper types used in queries` heading. I don't want to create a new module for 2 `newtypes`.  We can break up the module and group them appropriately once we are a little further along.